localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -281,6 +285,12 @@ void NDTScanMatcher::timer_diagnostic()
       diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::ERROR;
       diag_status_msg.message += "skipping_publish_num exceed limit. ";
     }
+    if (
+      state_ptr_->count("execution_time") &&
+      std::stod((*state_ptr_)["execution_time"]) >= critical_upper_bound_exe_time_ms_) {
+      diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::WARN;
+      diag_status_msg.message += "NDT exe time is too long. ";
```

Comment:
Would you mind adding the actual execution time as a reference here?

Reply:
Sorry I am not sure i understand your request well.

Reply:
What I meant is that the output would be better to be something like
```
NDT exe time is too long (took 39.4 [ms]).
```